### PR TITLE
[AIDAPP-132]: Additional Organization improvements

### PR DESCRIPTION
**Pull request by ketan-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/AIDAPP-132

### Technical Description

> Describe your changes in detail. This should include technical/architectural decisions and reasoning, if applicable.
> Changes done as asked in task description.
> Please ensure that at least one correct change type label is added to the PR. For example, if you are adding a new feature, please add the `Change Type | New Feature` label.

### Any deployment steps required?

> A deployment step could be a command that needs to be executed or an ENV key that needs to be added, for example.
>
> If yes, please describe the deployment steps required below and apply the `Deployment Steps` label to the PR.

_______________________________________________

### CODE REVIEW

File: database/migrations/2024_06_13_100154_data_seed_data_in_organization_types_table.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        $organization_types = DB::table('organization_types')->count();
+
+        if ($organization_types > 0) {
+            foreach ($this->organization_types_data as $record) {
+                DB::table('organization_types')
+                    ->where('name', $record['name'])
+                    ->delete();
+            }
+        }
+    }
```
Comment (Orrison) [inline database/migrations/2024_06_13_100154_data_seed_data_in_organization_types_table.php:90]: This causes 1 of the tests to fail. So lets add checks here to not run this migration during test

File: database/migrations/2024_06_13_100226_data_seed_data_in_organization_industries_table.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        $organization_industries = DB::table('organization_industries')->count();
+
+        if ($organization_industries > 0) {
+            foreach ($this->organization_industries_data as $record) {
+                DB::table('organization_industries')
+                    ->where('name', $record['name'])
+                    ->delete();
+            }
+        }
+    }
```
Comment (Orrison) [inline database/migrations/2024_06_13_100226_data_seed_data_in_organization_industries_table.php:120]: Same as the other migration lets not run this during tests

**Review by Orrison — CHANGES_REQUESTED** [review]

File: database/migrations/2024_06_13_100154_data_seed_data_in_organization_types_table.php (hunk 1 above)
Comment (ketan-canyon) [reply database/migrations/2024_06_13_100154_data_seed_data_in_organization_types_table.php]: resolved. sorry next time i will keep this things in mind.

File: database/migrations/2024_06_13_100226_data_seed_data_in_organization_industries_table.php (hunk 2 above)
Comment (ketan-canyon) [reply database/migrations/2024_06_13_100226_data_seed_data_in_organization_industries_table.php]: resolved. sorry next time i will keep this things in mind.

**Review by Orrison — APPROVED** [review]

**Pull request merged**